migrations/20231015013342-create-review.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('Reviews', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      title: {
+        type: Sequelize.STRING
+      },
+      description: {
+        type: Sequelize.TEXT
+      },
+      rating: {
+        type: Sequelize.INTEGER
+      },
+      movieId: {
+        type: Sequelize.INTEGER,
+        references:{
+          model:"Movie",
```

Comment:
looks good to me sebenernya eva, tp ini referencesnya tadi pas aku run gabisa, karena kamu di migration atas buatnya pake s Movies, tp disini referensinya Movie, nanti boleh di crosscheck aja yaa, 
gausah di fixing yg ini gapapa ttp nilai full ko buat di notes aja tugas berikutnya.. keep it up!! sisanya baguss